packages/jest-core/src/__tests__/TestScheduler.test.js

```diff
@@ -88,6 +88,7 @@ test('schedule tests run in parallel per default', async () => {
     context: {
       config: {
         runner: 'jest-runner-parallel',
+        transform: [],
```

Comment:
These should all be using `makeProjectConfig` :D

Reply:
yeah, agreed

Reply:
Done via #9250.

Seems usage of `GlobalConfig` is already pristine, so there's only a couple of tests that needed to be changed to use `makeProjectConfig`, most of which I'd already done so here.

I checked using some basic grepping for `as Config.ProjectConfig`, `projectConfig`, etc, but I'd be surprised if anything more complex would survive initial PR anyway.

I did notice that there's a bunch of `as Config.Argv`, most of which is `{} as Config.Argv` which I think can all be removed, but 🤷‍♀